stories/Molecules/Text/Faq/Faq.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { FaqFun } from './Faq_custom';
+import './Faq.scss';
+import { H5 } from '../../../Atom/Base-typography/Heading/h5';
+import { P } from '../../../Atom/Base-typography/Paragraph/p';
+
+export const Faq = ({headerText, descriptionText}) => {
+  useEffect(() => {
+    FaqFun();
+  })
+
+  return (
+   <div className="accordion" id="accordion">
+      <div className="accordion-head">
+         <H5 label={headerText} />
+         <div className="arrow down"></div>
+      </div>
+      <div className="accordion-body">
+         <P label={descriptionText} />
+         <P label={descriptionText} />
+         <P label={descriptionText} />
+      </div>
+      <div className="accordion-head">
+         <H5 label={headerText} />
+         <div className="arrow down"></div>
```

Comment:
@harishkumar-srijan Please check if we can use the arrow in heading :after

Reply:
Done